tutorials/plot_stats_cluster_erp.py

```diff
@@ -57,17 +57,15 @@
 # display the EEG data in Pandas format (first 5 rows)
 print(epochs.to_data_frame()[elecs].head())
 
-report = "{elec}, time: {tmin}-{tmax} msec; t({df})={t_val:.3f}, p={p:.3f}"
+report = "{elec}, time: {tmin}-{tmax} s; t({df})={t_val:.3f}, p={p:.3f}"
 print("\nTargeted statistical test results:")
 for (tmin, tmax) in time_windows:
+    A = long.to_data_frame(start=tmin, stop=tmax)
+    B = short.to_data_frame(start=tmin, stop=tmax)
```

Comment:
why not calling it tmin and tmax?

Reply:
There already are start and stop kwargs for to_data_frame, they are just ignored for input other than Raw.

Reply:
long.crop(tmin, tmax).to_data_frame()

would work no?

Reply:
Actually, `long.copy().crop(tmin, tmax).to_data_frame()`.

Sure, that would work. You think it's better?